controller/api/v1alpha1/servicedeployment_types.go

```diff
@@ -109,12 +158,35 @@ type ServiceSpec struct {
 	// Templated should apply liquid templating to raw yaml files, defaults to true
 	// +kubebuilder:validation:Optional
 	Templated *bool `json:"templated,omitempty"`
-
+	// +kubebuilder:validation:Optional
+	// +kubebuilder:validation:XValidation:rule="self == oldSelf",message="Imports are immutable"
+	Imports []corev1.ObjectReference `json:"imports"`
```

Comment:
I think to make this more future proof, we should have a type here like:

```
- stackRef:
      name: first-stack    
- stackRef:
      name: other-stack
```

That way we can add adjacent fields w/o breaking the schema

Reply:
Done.